### PR TITLE
TG-131: Admin Redux and shared components implementation.

### PR DESCRIPTION
**Pull request by gregorioelias**
**Redux**:
- [x] Create constants for admin entity.
- [x] Create actions for admin entity.
- [x] Create reducers for admin entity.
- [x] Create and implemented thunks for admin entity.

**Shared components**:
- [x] Implement Input from Shared Component
- [x] Implement Dropdown for "Active" from Shared Component
- [x] Implement Loading from Shared Component to handle fetching

### CODE REVIEW

**Comment by cocolobos** [comment]
Not working fine, I dont have the complete state of admin on the store

**Comment by cocolobos** [comment]
on EDIT_ADMIN_SUCCESS the prop isLoading is on true, should be false because is not loading anything

**Comment by cocolobos** [comment]
the actions dispatched on the form should be after pushing the confirm button of the Modal

**Comment by cocolobos** [comment]
After DELETE_ADMIN_SUCCESS we should dispatch the get all admins again to get the updated data from store

**Review by Lucas-DeLorenzi — APPROVED** [review]

**Review by Juanoauil — APPROVED** [review]

**Review by DeepFreq — APPROVED** [review]

File: src/Components/redux/modules/admins/reducer.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    case GET_ADMIN_SUCCESS:
+      return {
+        ...state,
+        list: action.payload,
+        isLoading: false
+      };
+    case GET_ADMIN_PENDING:
+      return {
+        ...state,
+        isLoading: true
```
Comment (marcomigliaro) [inline src/Components/redux/modules/admins/reducer.js:39]: Should set the error as empty by writing this:  error: initialState.error

File: src/Components/redux/modules/admins/reducer.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      return {
+        ...state,
+        list: [...state.list, action.payload],
+        isLoading: false
+      };
+    case POST_ADMIN_PENDING:
+      return {
+        ...state,
+        isLoading: true
+      };
```
Comment (marcomigliaro) [inline src/Components/redux/modules/admins/reducer.js:78]: Should set the error as empty by writing this:  error: initialState.error. Do this in all pending reducers

**Review by marcomigliaro — APPROVED** [review]

**Pull request merged**